plugins/attachment-resources/src/components/AttachmentRefInput.svelte

```diff
@@ -39,24 +43,61 @@
 
   const client = getClient()
   const query = createQuery()
+  const drafts = writable<Record<Ref<Doc>, Record<Ref<Attachment>, Attachment>>>()
```

Comment:
Please move out of component, so it could be reused, over-wise it will be alive only then component is alive.